samples/package.json

```diff
@@ -8,15 +8,28 @@
     "node": ">=8"
   },
   "repository": "googleapis/nodejs-asset",
-  "private": true,
```

Comment:
You want to keep that

Reply:
Can I know why? I think we want to make the sample code public?

Reply:
Sure!  So what `private` does in package.json is effectively make sure the package doesn't get `npm publish`'d.  This is important for us, because we don't want to accidentally publish this as a module:
https://docs.npmjs.com/files/package.json#private

The code will of course still be public and work :)